project-p1-first-webpage-master/code/index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My personal Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Julius+Sans+One&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Sunshiney&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Charmonman:wght@700&family=Sunshiney&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Charmonman:wght@700&family=Eagle+Lake&family=Sunshiney&display=swap" rel="stylesheet">
+    <!-- dont forget to add a css file and link it here! -->
+    <!-- If you're using google fonts, that should be added here -->
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <!-- 
+      Write your code here inside the body to make it appear in the browser. 
+      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if 
+      you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
+
+      <img src="example-recipe-background.png" alt="Banana Bread">
+      <img src="Page.JPG" alt="Me enjoying the view in Page, Arizona." width=100%>
+
+    -->
+  
+    <header>
+    <h1>Sabrina Ritz</h1>
+    <h2>teacher, translator, musician, composer, travel lover</h2>
+    </header>
+
+    <summary class="about-box">
+      <h2>About me</h2>
+      <p>Ich liebe es, immer wieder Neues zu lernen. I love exploring, traveling,... </p>
+    </summary>
+
+    <div class="flower-box">
+    <h2>What drives me</h2>
+    <img class="round-image" src="Blume.jpg" alt="Gelb weisse Blume mit fünf Blüten" width=300 height=300>
+    <div class="pelat_1">hhh</div>
+    <div class="pelat_2">hhh</div>
+    <div class="pelat_3">hhh</div>
+    <div class="pelat_4">hhh</div>
+    <div class="pelat_5">hhhhh</div>
```

Comment:
A suggestion would be to change the class names to for example `<pelat-one`, `pelat-two` etc. using kebab-case for CSS class names and also to change the corresponding names in the CSS-file.